ARSIS-Telemetry-GroundControl/telemetry-api/app/caches/user_cache.py

```diff
@@ -0,0 +1,36 @@
+class UserCache:
+    def __init__(self):
+        self.users = {}
+
+    def create_new_user_dict(self):
+        bpm = 100
+        o2 = 100
+        battery = 100
+        latitude = 10
+        longitude = 100
+        altitude = 0
+        heading = 0
+        return {
+            "biometrics": {"bpm": bpm, "o2": o2, "battery": battery},
+            "location": {
+                "latitude": latitude,
+                "longitude": longitude,
+                "altitude": altitude,
+                "heading": heading,
+            },
+        }
+
+    def get_all(self):
+        return self.users
+
+    def get(self, user_id):
```

Comment:
I'm not too python-savy... does anyone know if this has error handling or if it'll just crash? No error handling is fine for now ig, but I was just wondering.

Reply:
sry Its a little unclear, I'm referencing the "get" function

Reply:
As the `UserCache` is a data structure, the error should probably be handled in the code that calls `UserCache.get()`. I don't know the specifics of FastAPI, but I imagine that endpoints like `/location/{user}` would return a 5xx (probably 500) status code when the requested use is not found instead of the 404 that we would want. But it is probably fine for now.

Reply:
Yea that is a way better way to do it I was just trying to get something together to fix it.